build.sbt

```diff
@@ -11,10 +11,10 @@ val inputDirectory = Def.settingKey[File]("")
 lazy val encoders = project
   .in(file("encoders"))
   .settings(
-    libraryDependencies ++= Seq(
-      sparkSql,
-      munit % Test
-    )
+    libraryDependencies ++= Seq(sparkSql, munit % Test),
+    Test / parallelExecution := false,
+    // Test / fork := true,
+    // Test / javaOptions += "-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=1044"
```

Comment:
What are these settings?

Reply:
I use them to debug the tests in IntelliJ:
- Open the Spark project in IntelliJ
- run `sbt test`
- In IntelliJ, `Run > Attach To Process > Port 1044`

I prefer to leave it here for now because I will probably need it to implement the `Encoder[Array[X]]`.

Reply:
Sure, I did not know it was needed for Intellij

Reply:
You can start a debug process in IntelliJ  if you manage to import your project. It will use the same kind of option.

But I don't want to try to import `spark-scala3` in IntelliJ. So I start the process outside of it and then attach them.